onnxruntime/contrib_ops/cuda/bert/multihead_attention.h

```diff
@@ -33,6 +33,7 @@ class MultiHeadAttention final : public CudaKernel {
   bool disable_fused_cross_attention_;
   bool disable_flash_attention_;
   bool disable_memory_efficient_attention_;
+  bool enable_cudnn_flash_attention_;
```

Comment:
Should we change this to `disable_cudnn_flash_attention` once it is stable?

Reply:
The naming is fine since user can enable/disable in either choice of naming.

After more testing, we can change the default value to true, so that it is enabled by default in 1.20 release.
Currently, it is not enabled by default since we need more testing.